pages/cards/overview/big-arcana/index.js

```diff
@@ -0,0 +1,36 @@
+import Link from "next/link";
+import { useState, useEffect } from "react";
+import BackButton from "../../../../components/Backbutton/backbutton";
+import useStore from "../../../../src/store/store";
+
+export default function Overview() {
+  const [hasMounted, setHasMounted] = useState(false);
+
+  const cards = useStore((state) => state.allCards);
+
+  useEffect(() => {
+    setHasMounted(true);
+  }, []);
+  if (!hasMounted) {
+    return null;
+  }
```

Comment:
I don't understand this part, can you please explain to me why you need this?

Reply:
copy paste from another page where i was working with rng content - not needed here, as client and ssr aren't differing. thanks